File: jsprobelms/day10/com10.js
// compare the two values in array which is equal to 10
let arr=[1,23,4,5,6,3,7,10]
num=10
emp=[]
for(i=0;i<arr.length;i++){
    count=0
    for(j=i+1;j<arr.length;j++){
        if(arr[i]+arr[j]==10){
            emp.push(arr[i],arr[j])
            count++
            break

        }
    }
    // if(count>0){
    //     break
    // }
}
console.log(emp);

File: pythonproblems/day10/com10.py
# take an array return when two elements are equal to 10
arr=[1,2,3,4,5,6,7,8,10]
res=[]
for i in range(len(arr)):
    count=0
    for j in range(len(arr)):
        if arr[i]+arr[j]==10:
            res.append([arr[i],arr[j]])
            count+=1
            # break
    
    
print(res)